src/applications/avs/routes.jsx

```diff
@@ -1,16 +1,24 @@
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import PageNotFound from '@department-of-veterans-affairs/platform-site-wide/PageNotFound';
-
+import { useMyHealthAccessGuard } from '~/platform/mhv/hooks/useMyHealthAccessGuard';
 import ErrorBoundary from './components/ErrorBoundary';
 
 import Avs from './containers/Avs';
 
-const ErrorBoundaryWrapper = props => (
-  <ErrorBoundary>
-    <Avs {...props} />
-  </ErrorBoundary>
-);
+const ErrorBoundaryWrapper = props => {
+  const accessGuard = useMyHealthAccessGuard();
+
+  if (accessGuard) {
```

Comment:
I don't think this is necessary. The function will redirect automatically. Just call the function without the if